### PR TITLE
HAWQ-810. Add stringFormat for feature test library

### PR DESCRIPTION
**Pull request by xunzhang**

### CODE REVIEW

**Comment by xunzhang** [comment]
cc @zhangh43 @huor

**Comment by xunzhang** [comment]
also cc @jiny2

**Comment by jiny2** [comment]
+1 LGTM

File: src/test/feature/testlib/test_lib.cpp [hunk 1]
```diff
@@ -73,6 +74,11 @@ TEST_F(TestCommonLib, TestSqlUtil) {
   util.execSQLFile("testlib/sql/sample.sql", "testlib/ans/sample.ans");
 }
 
+TEST_F(TestCommonLib, TestStringFormat) {
+  auto s1 = hawq::test::stringFormat("welcome %s to apache %s project", "you", "HAWQ");
+  EXPECT_EQ(s1, "welcome you to apache HAWQ project");
```
Comment (huor) [inline src/test/feature/testlib/test_lib.cpp]:
```
auto s1 = hawq::test::stringFormat("Welcome to apache %s project", "HAWQ");
 +  EXPECT_EQ(s1, "Welcome to apache HAWQ project");
```
or
```
auto s1 = hawq::test::stringFormat("%s are welcome to apache %s project", "You", "HAWQ");
 +  EXPECT_EQ(s1, "You are welcome to apache HAWQ project");
```

**Comment by huor** [comment]
+1 except for the test case for string format.

File: src/test/feature/testlib/test_lib.cpp (hunk 1 above)
Comment (xunzhang) [inline src/test/feature/testlib/test_lib.cpp]: Here welcome is used as transitive verb. Refer webster: http://www.merriam-webster.com/dictionary/welcome

**Comment by xunzhang** [comment]
Merged into master.

**Pull request closed without merging**